pkg/signer/ecdsa_test.go

```diff
@@ -114,7 +119,16 @@ func TestSignature_VerifyFalse(t *testing.T) {
 	_, publicKey, _ := s.Sign(rawMessage[:])
 	sigToMarshal := ecdsaSignature{R: &big.Int{}, S: &big.Int{}}
 	m, _ := asn1.Marshal(sigToMarshal)
-	ok, err := Verify(rawMessage[:], m, UnmarshalKey(publicKey))
+	ecdsaPK, err := UnmarshalKey(publicKey)
+	require.NoError(t, err)
+	ok, err := Verify(rawMessage[:], m, ecdsaPK)
 	assert.False(t, ok)
 	assert.NoError(t, err)
 }
+
+func TestUnmarshalKey_Error(t *testing.T) {
+	ecdsaPK, err := UnmarshalKey([]byte(`wrongkey`))
+	require.Nil(t, ecdsaPK)
+	require.Error(t, err)
+	require.Equal(t, "invalid public key", err.Error())
```

Comment:
Once converted to an variable, please check if the error is returned with:

```suggestion
	require.ErrorIs(t, err, ErrInvalidPublicKey)
```

Reply:
@byo This suggestion is a bit incorrect: the 2nd param is the expected error, the actual one is the 3rd. Otherwise i agree - will change right now.

Reply:
Yup, The `ErrorIs` is a bit inconsistent with the rest of `testify`, I guess it's because they want to be consistend with the `errors.Is` signature.

Reply:
I've resolved all your suggestions.